.ado/jobs/integration-test.yml

```diff
@@ -117,7 +117,7 @@ jobs:
         inputs:
           artifactName: Targets - IntegrationTest-$(BuildPlatform)-$(System.JobAttempt)
           targetPath: packages/integration-test-app/windows/$(BuildPlatform)/Debug/integrationtest
-        condition: failed()
+        condition: always()
 
       - task: PublishPipelineArtifact@1
         displayName: Upload screenshots
```

Comment:
Was hoping to see the upload-build-logs.yml task but I'm not sure why that was skipped.... Maybe because yarn-integration tests still runs.. but fails ?!?!?

Reply:
Just changed that to `condition: always()` as well, although it should be running as is. Nothing after yarn-integration-test runs at the moment for some reason.

Reply:
Closing - #7546 addresses the issue this was getting at.